DSharpPlus/Entities/Guild/DiscordRoleTags.cs

```diff
@@ -0,0 +1,44 @@
+﻿using Newtonsoft.Json;
+
+namespace DSharpPlus.Entities
+{
+    /// <summary>
+    /// Represents a discord role tags.
+    /// </summary>
+    public class DiscordRoleTags
+    {
+        /// <summary>
+        /// Gets the id of the bot this role belongs to.
+        /// </summary>
+        [JsonProperty("bot_id", NullValueHandling = NullValueHandling.Ignore)]
+        public ulong? BotId { get; internal set; }
+
+        /// <summary>
+        /// Gets the id of the integration this role belongs to.
+        /// </summary>
+        [JsonProperty("integration_id", NullValueHandling = NullValueHandling.Ignore)]
+        public ulong? IntegrationId { get; internal set; }
+
+        /// <summary>
+        /// Gets whether this is the guild's premium subscriber role.
+        /// </summary>
+        [JsonIgnore]
+        public bool IsPremiumSubscriber
+        {
+            get
+            {
+                if(_premium_subscriber.HasValue)
+                {
+                    return _premium_subscriber.Value;
```

Comment:
Inconsistent code style. Braces and if/else belong in a new line each, and there should be a space between if and the parenthesis. Not to mention that if the value is unset this should be false I believe. And this can be shortened to `=> this._premiumSubscriber ?? false;` that way.

Also `_premium_subscriber` -> `_premiumSubscriber`.

Reply:
Thanks for that, it's the first time i work in C# so sorry for the wierd code style 😅

i will modify that

The value should return true if null because discord return a null value if present and nothing if not. (according to [the docs](https://discord.com/developers/docs/topics/permissions#role-object-role-tags-structure))

Reply:
i've double checked and it should be good now